### PR TITLE
Add translations of ja_JP, zh_CN and zh_TW.

### PR DESCRIPTION
**Pull request by qianxu2001**

### CODE REVIEW

File: po/zh_CN/ftpd.po [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+msgstr "日志"
+
+#: /home/mtheall/workspace/switch/ftpd/source/ftpServer.cpp:346
+msgid "Menu"
+msgstr "菜单"
+
+#. TRANSLATORS: \xee\x80\x83 (Y button)
+#: /home/mtheall/workspace/switch/ftpd/source/ftpServer.cpp:344
+msgid "Menu "
+msgstr "菜单 "
```
Comment (LitoMore) [inline po/zh_CN/ftpd.po:88]: What is this ` `?

There is a ` \ue003` after `菜单`.

Comment (qianxu2001) [reply po/zh_CN/ftpd.po:88]: The original text is ``, I just kept it during translation. as the remark says, that is Y button.

**Review by LitoMore — APPROVED** [review]

**Comment by mtheall** [comment]
Can you give some insight on the demographic that uses zh_JP?

**Comment by qianxu2001** [comment]
> Can you give some insight on the demographic that uses zh_JP?

Zh_JP is added because zh_CN and zh_TW cannot be used directly, because the system font is not complete and garbled characters are displayed. zh_JP is actually based on zh_TW, and replaces the characters that cannot be displayed with similar characters in the Japanese font. In this way, the text can be displayed completely without installing a third-party font library. Both zh_CN and zh_TW need to install third-party fonts.

File: po/zh_CN/ftpd.po (hunk 1 above)
Comment (LitoMore) [reply po/zh_CN/ftpd.po:88]: Soga. Thanks!

**Comment by fincs** [comment]
Friendly reminder that both 3DS and Switch ship with *all* system fonts regardless of region, and it is possible to load the system font of *any* region on *any* console (including Traditional/Simplified Chinese fonts): [3ds citro2d](https://github.com/devkitPro/citro2d/blob/master/source/font.c#L173-L195), [Switch libnx](https://github.com/switchbrew/libnx/blob/master/nx/include/switch/services/pl.h#L48)

**Pull request closed without merging**